build_projects/update-dependencies/UpdateFilesTargets.cs

```diff
@@ -87,7 +87,14 @@ public static BuildTargetResult ReplaceProjectJson(BuildTargetContext c)
         {
             List<DependencyInfo> dependencyInfos = c.GetDependencyInfos();
 
-            IEnumerable<string> projectJsonFiles = Directory.GetFiles(Dirs.RepoRoot, "project.json", SearchOption.AllDirectories);
+            IEnumerable<string> projectJsonFiles = new string[]
```

Comment:
Hmm, well, it's just picking some paths rather than recursively finding all, so it shouldn't seem like a hack. I wrote it in a confusing way, though, seeing it again... I'll change it to a `List<string>` and see if that helps.